__tests__/app.test.js

```diff
@@ -62,9 +59,60 @@ describe("TOPICS", () => {
       .then(({body}) => {
       expect(body.endpoints).toEqual(endpoints)
   
-
       })
     })
   })
+});
+
+//ARTICLES////////////////////
+
+describe("ARTICLES", () => {
+
+  describe("/api/articles/:article_id", () => {
+
+
+    test("Returns 200 status code and the correct article when given valid article ID", () => {
+      return request(app)
+      .get('/api/articles/1')
+      .expect(200)
+      .then((response) => {
+ 
+      const article = response.body.article
+      expect(article).toEqual( {
+        
```

Comment:
I would use a flexible matcher like toMatchObject or objectContaining as if there were any changes to the article this would then fail